docs/source/precalculations/text_metadata.rst

```diff
@@ -0,0 +1,104 @@
+.. _text_metadata:
+
+Text Metadata Precalculation
+============================
+
+Description
+-----------
+
+This precalculation computes metadata about a text. 
+Since many checks cannot work with textual columns, this precalculation is used to numerize and categorize the texts.
+It generates new columns with data about the textual column, such as:
+
+NumCharsMetadata
+    Returns the total number of characters in the text.
```

Comment:
I'm not really satisfied with the format you are using here. In the output it looks a little bit like copied from a text editor to word or similar. Perhaps @L-v-M has an idea how to make this look fancier. It seems like there are some fancy methods that can do such things. (look API documentation)